manifests/plugin/discovery.pp

```diff
@@ -0,0 +1,46 @@
+# = Foreman Proxy Discovery plugin
+#
+# This class installs discovery plugin and images on smart proxys
+#
+# === Parameters:
+#
+# $install_images::  should the installer download and setup discovery images
```

Comment:
I'd prefer leaving out the installer and have a more generic description like "should discovery images be downloaded and setup?"